js/views/navigation-accounts.js

```diff
@@ -49,7 +49,9 @@ define(function(require) {
 				});
 			});
 
-			folder.set('active', true);
```

Comment:
small bugfix: if the first folder fails loading, `folder` was undefined here and therefore an error was shown in the js console.